docs/source/modin/general_functions.rst

```diff
@@ -2,7 +2,7 @@
 General functions
 =============================
 
-.. currentmodule:: snowflake.snowpark.modin.pandas.general
```

Comment:
@sfc-gh-azhan are you reviewing modin related changes?

Reply:
Yes let me do it. This PR should not have those modin changes. All changes under `ci`, `docs/source`, `modin` should be rolled back.